Leaflet/index.html

```diff
@@ -126,15 +126,9 @@ <h1>Leaflet with Bird.i imagery</h1>
 		}
 
 		let catalogUrl =
-			"https://catalog-api.hibirdi.com/timeseries/lat/" + lat.toFixed(6) +
+			"https://catalog-api.hibirdi.com/catalog/lat/" + lat.toFixed(6) +
```

Comment:
As another nifty javascript trick, you might want to use a string template here instead of string concatenation (ESLint usually throws up a warning for this) (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)